[ci] disable flaky MobileProfiler.Backend test

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #78320

This test is flaky, normally I'd disable using the disable bot but it
doesn't support cpp.

[skip ci]

suo committed:
[ci] disable flaky MobileProfiler.Backend test

This test is flaky, normally I'd disable using the disable bot but it
doesn't support cpp.

[skip ci]

[ghstack-poisoned]

suo commented:
tracking fixing and re-enabling at: https://github.com/pytorch/pytorch/issues/78283

malfet commented on test/cpp/lite_interpreter_runtime/test_mobile_profiler.cpp:
```diff
@@ -70,38 +70,38 @@ TEST(MobileProfiler, ModuleHierarchy) {
   ASSERT_TRUE(checkMetaData("aten::add", metadata_name, "top(C)::<unknown>.aten::add", trace_file));
 }
 
-TEST(MobileProfiler, Backend) {
```
One can skip tests by adding `DISABLED_` prefix to its name

malfet reviewed (approved):
LGTM, but please consider disabling using `DISABLED_` prefix akin to https://github.com/pytorch/pytorch/blob/7ea5fa3dd4c544f2508c5697f103d4ef8f183ed8/test/cpp/tensorexpr/test_loopnest.cpp#L2207

suo committed:
Update on "[ci] disable flaky MobileProfiler.Backend test"


This test is flaky, normally I'd disable using the disable bot but it
doesn't support cpp.

[skip ci]

[ghstack-poisoned]

suo commented:
@pytorchbot merge -f

suo committed:
Update on "[ci] disable flaky MobileProfiler.Backend test"


This test is flaky, normally I'd disable using the disable bot but it
doesn't support cpp.

[skip ci]

[ghstack-poisoned]

suo commented:
@pytorchbot merge this

suo commented:
@pytorchbot merge -f
